File: mobile/src/main/java/net/maiatoday/levelbest/di/ApplicationComponent.java
package net.maiatoday.levelbest.di;

import javax.inject.Singleton;

import dagger.Component;

/**
 * Application Component for dependency injection
 * Created by maia on 2016/07/25.
 */

@Singleton
@Component(modules = {ApplicationModule.class})
public interface ApplicationComponent extends LBComponent {
}

File: mobile/src/main/kotlin/net/maiatoday/levelbest/di/ApplicationComponent.kt
package net.maiatoday.levelbest.di

import javax.inject.Singleton

import dagger.Component

/**
 * Application Component for dependency injection
 * Created by maia on 2016/07/25.
 */

@Singleton
@Component(modules = arrayOf(ApplicationModule::class))
interface ApplicationComponent : LBComponent
